### PR TITLE
bpo-26128: Added __init__to STARTUPINFO

### PR DESCRIPTION
**Pull request by codenamesubho**

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately our records indicate you have not signed the CLA. For legal reasons we need you to sign this before we can look at your contribution. Please follow these steps to rectify the issue:

1. Sign the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/)
2. **Wait at least one US business day** and then check "Your Details" on [bugs.python.org](https://bugs.python.org/) to see if your account has been marked as having signed the CLA (the delay is due to a person having to manually check your signed CLA)
3. Reply here saying you have completed the above steps


Thanks again to your contribution and we look forward to looking at it!

File: Lib/subprocess.py [hunk 1]
```diff
@@ -127,11 +127,13 @@ def stdout(self, value):
     import msvcrt
     import _winapi
     class STARTUPINFO:
-        dwFlags = 0
-        hStdInput = None
-        hStdOutput = None
-        hStdError = None
-        wShowWindow = 0
+        def __init__(self,dwFlags=0,hStdInput=None,hStdOutput=None,
```
Comment (sayanchowdhury) [inline Lib/subprocess.py]: This needs to be implemented via keyword only arguments as the Terry suggested in the issue.

Comment (sayanchowdhury) [reply Lib/subprocess.py]: .. and spaces after the args

**Review by sayanchowdhury — CHANGES_REQUESTED** [review]

**Comment by ncoghlan** [comment]
OK, I've figured out how to push updates back to PR branches (hence the rebase atop the master branch), so next step will be to add the bookkeeping updates (ACKS, NEWS, What's New) and then merge the change.

**Comment by ncoghlan** [comment]
OK, I've figured out how to push updates back to PR branches (hence the rebase atop the master branch), so next step will be to add the bookkeeping updates (ACKS, NEWS, What's New) and then merge the change.

**Comment by codenamesubho** [comment]
> so next step will be to add the bookkeeping updates (ACKS, NEWS, What's New) and then merge the change.

@ncoghlan , Anymore changes I need to do for this patch?

**Comment by codenamesubho** [comment]
@ncoghlan Added Acks and News, wasn't sure what to write in "What's New". Letme know if anything is missing. Thanks.

**Comment by ncoghlan** [comment]
For library changes, the What's New guide is generally just a feature-focused summary of the changes added in Misc/NEWS. However, given how conflict-prone Misc/NEWS is at the moment, I went ahead and merged this without the What's New changes (there's generally a review pass through Misc/NEWS around the beta phase to look for feature additions that aren't mentioned in What's New yet).

**Comment by ncoghlan** [comment]
Added the sprint label, as this PR was submitted at the PyCon Pune 2017 core development sprint.

**Pull request merged**